share/composer/openstack.ks

```diff
@@ -0,0 +1,46 @@
+# Lorax Composer openstack output kickstart template
+
+# Firewall configuration
+firewall --enabled
```

Comment:
This should change to:

```
firewall --disabled
```

Openstack has its own firewall, and this gets in the way.